generator/src/main/java/com/scottlogic/deg/generator/validators/GenerationConfigValidator.java

```diff
@@ -19,6 +32,42 @@ public ValidationResult validateCommandLine(GenerationConfig config) {
             errorMessages.add("RANDOM mode requires max row limit: use -n=<row limit> option");
         }
 
+        if (!configSource.overwriteOutputFiles()) {
+            if (configSource.shouldViolate()) {
+                checkViolationGenerateOutputTarget(errorMessages, outputTarget);
+            } else {
+                checkGenerateOutputTarget(errorMessages, outputTarget);
+            }
+        }
+
         return validationResult;
     }
+
+    private void checkGenerateOutputTarget(ArrayList<String> errorMessages, OutputTarget outputTarget) {
+        if (outputTarget.isDirectory()) {
+            errorMessages.add("Invalid Output - target is a directory, please use a different output filename or use the --overwrite option");
```

Comment:
`--overwrite` isn't applicable here, it must be a file.

Reply:
ah, yes, fixed it.